### DISCUSSION TITLE
WWFF search criteria adjustment in Quick lookup

### DISCUSSION THREAD

**Discussion by oh1jw** [magicbug/Cloudlog#959] [General]
Hey,
i have been in a hurry and only now have time to test this feature properly. Thanks for the great feature for WWFF-operators. 

It turned out that in the search you must also enter the reference number in the search for WWFF connections (eg OHFF-0808). Otherwise, there are no results for the search. 

Is it possible to change that search so that it only retrieves the SIG field with WWFF data and ignores its reference number in the SIG INFO field? Or alternatively use the truncated search with the *-symbol?
![image](https://user-images.githubusercontent.com/77230101/111885252-590a1280-89cf-11eb-919a-d9f80f0e2dd4.png)

And if you want to search for something specific WWFF target, you could ad it's ref. number.
![image](https://user-images.githubusercontent.com/77230101/111885367-f7967380-89cf-11eb-80ce-5b3a4a9b2418.png)

-Juuso

**Comment by AndreasK79** [comment]
That defeats the whole purpose of the Quick lookup. The whole purpose is to quickly be able to see if you have worked something on a band and mode. If a wildcard (*) search is implemented, then a W or C will be marked everywhere you have worked a WWFF, but you will not know which reference you have worked.

**Reply by oh1jw** [reply]
It works differently than I thought it would work, but **it works just as it actually makes sense to act**. Thanks for clarifying the idea. Thanks @AndreasK79

**Reply by AndreasK79** [reply]
No problem @oh1jw. This new feature can be expanded, so feel free to suggest things. Perhaps you were more after a list of all refs worked?

**Reply by oh1jw** [reply]
Yes, but it is just as important to quickly know if you have already worked that ref.

**Reply by AndreasK79** [reply]
That was the original idea behind the Quick lookup, but I just got an idea. If you enter * and click show, show a list of worked refs instead. Would that be of interest?

**Reply by oh1jw** [reply]
> That was the original idea behind the Quick lookup, but I just got an idea. If you enter * and click show, show a list of worked refs instead. Would that be of interest?

Is it possible that it has both features? Quick check of a specific reference and then all listed with *.

**Reply by AndreasK79** [reply]
I think it should be possible. What kind of information would you like in a list like that?

**Reply by oh1jw** [reply]
> I think it should be possible. What kind of information would you like in a list like that?

**Actually**, there is no need for that kind of listing, because I just saw that the Awards - SIG generates the list. There is no need for double that in quick lookup. Sorry - let's keep the Quick lookup simple as you had originally thought. 

My bad - we can close this thread.
